hrqb/tasks/employee_salary_history.py

```diff
@@ -144,13 +151,51 @@ def _set_base_salary_change_percent(self, salary_df: pd.DataFrame) -> pd.DataFra
                 / new_salary_df["previous_base_amount"]
                 - 1.0
             ),
-            3,
+            PERCENT_DECIMAL_ACCURACY,
         )
         new_salary_df["base_change_percent"] = new_salary_df["base_change_percent"].where(
             new_salary_df["previous_base_amount"].notna(), 0.0
         )
         return new_salary_df
 
+    def _set_effective_salary_and_change_percent(
```

Comment:
This is _very_ similar to the method `_set_base_salary_change_percent` defined above: calculating a change percentage based on previous rows.  

It _almost_ lends itself to abstracting out to a shared method where you pass fields in the dataframe... but likely only these two, so probably too much overhead for the gain.